Makefile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run unit tests
+test:
+	python3 -m unittest unit_test/k6/test_renderer.py
+
+# Build howitzer docker images
+build.howitzer:
+	docker build -t k6_test:dev -f perf_test/docker/Dockerfile.slim .
```

Comment:
I think this is okay for now? When I work on the CI part of this for wml-serving, will probably pull the tag out as a variable. But this will just build a Docker image locally with this name which is fine!